docs/topics/Rate_Limits.md

```diff
@@ -50,15 +50,15 @@ Note that the normal rate-limiting headers will be sent in this response. The ra
 ```
 < HTTP/1.1 429 TOO MANY REQUESTS
 < Content-Type: application/json
-< Retry-After: 6457
+< Retry-After: 65
 < X-RateLimit-Limit: 10
 < X-RateLimit-Remaining: 0
 < X-RateLimit-Reset: 1470173023
 < X-RateLimit-Reset-After: 7
```

Comment:
I think you forgot to update this header to `64.57`. Possibly also the one above that (to something like `X-RateLimit-Reset: 1470173023.420`)

Reply:
From the other comments, I believe this is rounded up.

[RFC-7231 S7.1.3 specifies that the value for a `Retry-After` header should be an integral type](https://tools.ietf.org/html/rfc7231#section-7.1.3), so changing this to a decimal value would make this API not comply to _Hypertext Transfer Protocol (HTTP/1.1): Semantics and Content_.

Reply:
I was referring to `X-RateLimit-Reset-After`; `Retry-After` is indeed supposed to be in rounded settings.